l10n_fi_payment_reference/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+The module allows users to generate a valid invoice payment reference for sales invoices.
```

Comment:
Please update to match the current implementation. The payment reference is already there, so this description doesn't really match the contents of this module.

Reply:
I updated description